### PR TITLE
📖 [bento][amp-accordion] Update bento documentation to match template

### PR DESCRIPTION
**Pull request by krdwan**
Update bento docs to match template.

Referenced template PR: https://github.com/ampproject/amphtml/pull/31944

### CODE REVIEW

File: extensions/amp-accordion/amp-accordion.md [hunk 1]
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
...
-  api.toggle('section1');
-</script>
 ```
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-accordion-1.0.css">
+```
+
+Fully valid AMP pages use the AMP layout system to infer sizing of elements to create a page structure before downloading any remote resources. However, Bento use imports components into less controlled environments and AMP's layout system is inaccessible.
 
-[/filter] <!-- formats="websites" -->
+## Version notes
```
Comment (caroqliu) [inline extensions/amp-accordion/amp-accordion.md:286]: Should this go to the bottom of the file?

File: extensions/amp-accordion/amp-accordion.md [hunk 2]
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
...
+    // set up button actions
+    document.querySelector('#button1').onclick = () => api.expand();
+    document.querySelector('#button2').onclick = () => api.expand('section1');
+  })();
+</script>
 ```
 
-### collapse
-
+**collapse**
```
Comment (CrystalOnScript) [inline extensions/amp-accordion/amp-accordion.md:254]:
```suggestion
**collapse()**
```

File: extensions/amp-accordion/amp-accordion.md [hunk 3]
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
...
+    document.querySelector('#button2').onclick = () => api.toggle('section1');
+  })();
+</script>
 ```
 
-[/example]
-
-### expand
-
+**expand**
```
Comment (CrystalOnScript) [inline extensions/amp-accordion/amp-accordion.md:233]:
```suggestion
**expand()**
```

File: extensions/amp-accordion/amp-accordion.md [hunk 4]
```diff
@@ -95,118 +95,111 @@ The example below contains an `amp-accordion` with three sections. The
...
+    // when section 2 collapses, section 1 also collapses
+    const section1 = document.querySelector('#section1');
+    const section2 = document.querySelector('#section2');
+    section1.addEventListener('expand', () => api.expand('section2'));
+    section2.addEventListener('collapse', () => api.collapse('section1'));
+  })();
+</script>
+```
 
+**toggle**
```
Comment (CrystalOnScript) [inline extensions/amp-accordion/amp-accordion.md:197]:
```suggestion
**toggle()**
```

File: extensions/amp-accordion/amp-accordion.md (hunk 3 above)
Comment (caroqliu) [inline extensions/amp-accordion/amp-accordion.md:233]:
```suggestion
**expand()**
```

File: extensions/amp-accordion/amp-accordion.md (hunk 2 above)
Comment (caroqliu) [inline extensions/amp-accordion/amp-accordion.md:254]:
```suggestion
**collapse()**
```

**Review by CrystalOnScript — CHANGES_REQUESTED** [review]
This looks nearly perfect! Just a few nits and one re-order request:

Could you move the actions above the events and add a header above each?
```
##### Actions

actions....

##### Events

events...
```

Similar to amp-lightbox.md in this pr
https://github.com/ampproject/amphtml/commit/c1d09c457911f5906debeb04a5f0c14bf8734618

File: extensions/amp-accordion/amp-accordion.md (hunk 4 above)
Comment (caroqliu) [inline extensions/amp-accordion/amp-accordion.md:197]:
```suggestion
**toggle()**
```

**Comment by krdwan** [comment]
> This looks nearly perfect! Just a few nits and one re-order request:
> 
> Could you move the actions above the events and add a header above each?
> 
> ```
> ##### Actions
> 
> actions....
> 
> ##### Events
> 
> events...
> ```
> 
> Similar to amp-lightbox.md in this pr
> [c1d09c4](https://github.com/ampproject/amphtml/commit/c1d09c457911f5906debeb04a5f0c14bf8734618)

Thanks for the review, I've re-arranged per your comments and addressed nits!!

File: extensions/amp-accordion/amp-accordion.md [hunk 5]
```diff
@@ -95,118 +95,111 @@ The example below contains an `amp-accordion` with three sections. The
...
-Include the `animate` attribute in `<amp-accordion>` to add a "roll down"
-animation when the content is expanded and "roll up" animation when collapsed.
-
-[example preview="top-frame" playground="true" imports="amp-accordion:1.0"]
+The example below demonstrates `amp-accordion` component in standalone use.
 
 ```html
-<amp-accordion animate>
-  <section>
+<head>
```
[the remark this answers is not in the document]
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md:109]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 2 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 3 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 4 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 3 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 2 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 4 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, updated!

File: extensions/amp-accordion/amp-accordion.md (hunk 1 above)
Comment (krdwan) [reply extensions/amp-accordion/amp-accordion.md]: Thanks, moved to the bottom of the file!

**Review by caroqliu — APPROVED** [review]

**Review by CrystalOnScript — APPROVED** [review]

**Pull request merged**